package.json

```diff
@@ -68,19 +68,23 @@
         "build": "webpack-cli --mode development && node pugconfig.js",
         "build-watch": "webpack-cli --mode production --watch",
         "start": "http-server ./build -a localhost -c-1",
-        "start-watch": "run-p build-watch pug-watch start",
+        "start-watch": "run-p build-watch pug-watch demos-watch start",
         "build-dist": "webpack-cli --mode production --config webpack.config.dist.js",
         "build-lib": "webpack-cli --mode production --config webpack.config.lib.js",
-        "pug-watch": "node pugconfig.js --watch",
+        "demos": "webpack-cli --mode production --config webpack.config.demos.js",
+        "demos-watch": "webpack-cli --mode production --config webpack.config.demos.js --watch",
         "cleanup": "rimraf ./dist ./lib ./build",
         "lint": "tslint --project tsconfig.json --format stylish --config tslint.json",
         "test": "mocha",
         "test-nyc": "nyc mocha",
         "test-watch": "mocha --watch",
         "coverage": "nyc report --reporter=text-lcov | coveralls",
         "doc": "typedoc source/webgl-operate.ts",
-        "ci": "run-p build-dist build-lib lint test-nyc",
+        "pug": "node pugconfig.js",
+        "pug-watch": "node pugconfig.js --watch",
+        "ci": "run-p build-dist build-lib lint test-nyc website",
```

Comment:
I'm not sure about running `test-nyc` and `website` indirectly when npm-installing from git (via `prepack` and `preversion`). Takes quite some extra time though only `build-dist` and `build-lib` are required in that case.